app/uk/gov/hmrc/fileupload/controllers/envelope.scala

```diff
@@ -73,15 +73,40 @@ object CreateEnvelopeRequest {
   implicit val constraintsFormats: OFormat[EnvelopeConstraintsUserSetting] = Json.format[EnvelopeConstraintsUserSetting]
   implicit val formats: OFormat[CreateEnvelopeRequest] = Json.format[CreateEnvelopeRequest]
 
-  def formatUserEnvelopeConstraints(constraintsO: EnvelopeConstraintsUserSetting): Option[EnvelopeConstraints] = {
-    Some(EnvelopeConstraints(maxItems = constraintsO.maxItems.getOrElse(defaultMaxItems),
-                             maxSize = constraintsO.maxSize.getOrElse(defaultMaxSize).toUpperCase(),
-                             maxSizePerItem = constraintsO.maxSizePerItem.getOrElse(defaultMaxSizePerItem).toUpperCase(),
-                             contentTypes = checkContentTypes(constraintsO.contentTypes.getOrElse(defaultContentTypes))
+  def formatUserEnvelopeConstraints(constraintsO: EnvelopeConstraintsUserSetting,
```

Comment:
This method also can be extracted, even to same file as first method I've shown you.

Reply:
thanks, has done